CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- n/a
+### Fixed
+- Fixed minor bug in map_404_to_none, where the response could have response equals to None
+- rewrite set-maintenace example, make it more realistic
```

Comment:
I don't think example changes need to be mentioned in changelog at all, because they're not included in releases.  (Examples don't go into the source tarball uploaded to pypi.)